cst/plugins/eu.esdihumboldt.cst.functions.numeric/src/eu/esdihumboldt/cst/functions/numeric/sequentialid/SequentialID.java

```diff
@@ -78,16 +79,21 @@ protected Object evaluate(String transformationIdentifier, TransformationEngine
 			break;
 		}
 
-		long id;
+		long id = 1;
 		synchronized (context) {
 			Long seqValue = (Long) context.get(key);
 			if (seqValue != null) {
 				id = seqValue + 1;
 			}
 			else {
-				id = START_VALUE;
+				try {
+					id = Integer.parseInt(startValue);
+					context.put(key, id);
+				} catch (Exception E) {
+					System.out.println("Error with the input value of " + startValue
```

Comment:
I would rather be for letting this fail rather than catching the error and continuing.

I think it should work to throw an exception here with an adapted message and the original exception as cause.